pkg/vfs/errors.go

```diff
@@ -35,4 +35,6 @@ var (
 	ErrDirNotEmpty = errors.New("Directory is not empty")
 	// ErrWrongCouchdbState is given when couchdb gives us an unexpected value
 	ErrWrongCouchdbState = errors.New("Wrong couchdb reduce value")
+	// ErrFileTooBig is used when there is no more space left on the filesystem
+	ErrFileTooBig = errors.New("The file is too big")
```

Comment:
May be we can make this error a bit more clear "Adding file would exceed quota", as it is now it makes me think more of 413 error, aka "we will never accept a file bigger than 10Go"